# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/win/src/interception_internal.h
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 3
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/sandbox/win/src/interception_internal.h
--- a/security/sandbox/chromium/sandbox/win/src/interception_internal.h	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/sandbox/win/src/interception_internal.h	Wed Jun 12 11:10:48 2019 +0100
@@ -28,21 +28,21 @@
 // for the InterceptionAgent.
 // A single interception:
 struct FunctionInfo {
-  size_t record_bytes;            // rounded to sizeof(size_t) bytes
+  size_t record_bytes;  // rounded to sizeof(size_t) bytes
   InterceptionType type;
   InterceptorId id;
   const void* interceptor_address;
-  char function[1];               // placeholder for null terminated name
+  char function[1];  // placeholder for null terminated name
   // char interceptor[]           // followed by the interceptor function
 };
 
 // A single dll:
 struct DllPatchInfo {
-  size_t record_bytes;            // rounded to sizeof(size_t) bytes
+  size_t record_bytes;  // rounded to sizeof(size_t) bytes
   size_t offset_to_functions;
   int num_functions;
   bool unload_module;
-  wchar_t dll_name[1];            // placeholder for null terminated name
+  wchar_t dll_name[1];  // placeholder for null terminated name
   // FunctionInfo function_info[] // followed by the functions to intercept
 };
 
@@ -50,7 +50,7 @@
 struct SharedMemory {
   int num_intercepted_dlls;
   void* interceptor_base;
-  DllPatchInfo dll_list[1];       // placeholder for the list of dlls
+  DllPatchInfo dll_list[1];  // placeholder for the list of dlls
 };
 
 // Dummy single thunk:
@@ -65,7 +65,7 @@
   void* base;
   int num_thunks;
 #if defined(_WIN64)
-  int dummy;                      // Improve alignment.
+  int dummy;  // Improve alignment.
 #endif
   ThunkData thunks[1];
 };